plugins/available/gitstatusd.plugin.bash

```diff
@@ -0,0 +1,22 @@
+cite about-plugin
+about-plugin 'speeds up your life by using gitstatusd for git status calculations. install from https://github.com/romkatv/gitstatus'
+
+
+function gitstatusd_on_disable() {
+  about 'Destructor of gitstatud plugin'
+  group 'gitstatud'
+
+  unset SCM_GIT_USE_GITSTATUSD
+}
+
+[[ "$SCM_CHECK" == "true" ]] || return # No scm-check
+
+[[ $- == *i* ]] || return  # non-interactive shell
+
+[[ ! -z "${SCM_GIT_GITSTATUSD_LOC}" ]] || SCM_GIT_GITSTATUSD_LOC="$HOME/gitstatus/gitstatus.plugin.sh"
```

Comment:
This is usually done like this:

```sh
: "${SCM_GIT_GITSTATUSD_LOC:=$HOME/gitstatus/gitstatus.plugin.sh}"
```

You might also want to rename this parameter as it doesn't actually point to gitstatusd.

Reply:
done and renamed